test/cases/basic.html

```diff
@@ -1,3 +1,3 @@
 <p>Testing of the target blank extension.</p>
 
-<p>Link to <a target="_blank" href="http://google.com">Google</a></p>
\ No newline at end of file
+<p>Link to <a target="_blank" target="_blank" href="http://google.com">Google</a></p>
```

Comment:
Since this broken test could be due to a showdown bug, I would let this line unchanged and and see if the showdown author can fix the bug instead of we changing the test.